rebasehelper/tests/conftest.py

```diff
@@ -70,21 +71,26 @@ def mocked_spec_object(spec_attributes):
     spec = SpecFile.__new__(SpecFile)
     spec.save = lambda: None
     for attribute, value in spec_attributes.items():
+        if attribute == 'macros':
+            for macro, properties in value.items():
+                rpm.addMacro(macro, properties.get('value', ''))
+            macros = MacroHelper.dump()
+            for macro, properties in value.items():
+                for m in macros:
+                    if m['name'] == macro:
+                        for prop, v in properties.items():
+                            if prop != 'value':
+                                m[prop] = v
+            value = macros
```

Comment:
Why this move? I mean, why not do it this way already in commit 0e7473d?

Reply:
Hmm, I amended to a wrong commit :/ I didn't really like setting the `macros` attribute to spec (which aren't really the macros intended for use) and wanted to modify the attribute correctly inside the `for` loop, to stay consistent with how `SpecContent` is handled.